sm_annotation_utils/sm_annotation_utils.py

```diff
@@ -24,17 +24,19 @@ def _extract_data(res):
 DEFAULT_CONFIG = {
     'graphql_url': 'http://metaspace2020.eu/graphql',
     'moldb_url': 'http://metaspace2020.eu/mol_db/v1',
-    'jwt': None
+    'jwt': 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJNRVRBU1BBQ0UyMDIwIiwicm9sZSI6ImFub255bW91cyJ9.Hl0h6crcHLb-SPm7nomXkQco5l2iAO6D1bwdjmOaFXM'
```

Comment:
I'd guess hardcoding JWTs is a bit fragile (I can imagine it could be expired without warning) @LachlanStuart & @intsco - any suggestions for a longer term solution?

Reply:
It is, and this wouldn't even be necessary if it weren't for a bug that made it into production. My original intent when writing the access control code was to treat not having a JWT as the same as having an anonymous JWT, but I missed checking for nulls in 1 place :( I've pushed a fix for sm-graphql, but it won't go to production until the next release.

My plan is to move the responsibility of JWT issuing to sm-graphql and add a username/password system so that JWTs can be requested directly. This should happen as part of Private Space, as we need to move the Users database into Postgres as part of those changes and it's easier to move the login functionality into sm-graphql than to deal with sm-webapp needing Postgres access.